app/bundles/CoreBundle/Controller/AbstractStandardFormController.php

```diff
@@ -342,7 +348,7 @@ protected function deleteStandard($objectId)
      *
      * @throws \Exception
      */
-    protected function editStandard($objectId, $ignorePost = false)
+    protected function editStandard($objectId, $ignorePost = false, string $engine = self::ENGINE_PHP)
```

Comment:
@dennisameling I'm wondering whether we can avoid this somehow as it is a temporary change until we get rid of the PHP templates completely in M5 (in few months). How about to temporarily modify the `getTemplateName()` method from
```php
    protected function getTemplateName($file)
    {
        if ($this->get('templating')->exists($this->getControllerBase().':'.$file)) {
            return $this->getControllerBase().':'.$file;
        } elseif ($this->get('templating')->exists($this->getTemplateBase().':'.$file)) {
            return $this->getTemplateBase().':'.$file;
        } else {
            return 'MauticCoreBundle:Standard:'.$file;
        }
    }
```
to ```php
    protected function getTemplateName($file, string $engine = self::ENGINE_TWIG)
    {
        if ($engine === self::ENGINE_TWIG && strpos($file, '.php')) {
            $file = str_replace('.php', '.twig', $file);
        }
        if ($this->get('templating')->exists($this->getControllerBase().':'.$file)) {
            return $this->getControllerBase().':'.$file;
        } elseif ($this->get('templating')->exists($this->getTemplateBase().':'.$file)) {
            return $this->getTemplateBase().':'.$file;
        } elseif  ($this->get('templating')->exists('MauticCoreBundle:Standard:'.$file)) {
            return 'MauticCoreBundle:Standard:'.$file;
        } elseif ($engine === self::ENGINE_TWIG) {
            return $this->getTemplateName($file, self::ENGINE_PHP);
        }

        throw new \Exception("Template {$file} not found");
    }
```
The idea is to search for twig templates first and fallback to php templates if not found. This way we won't need to modify controller method params. It will be a bit slower but only for templates still using PHP which should be resolved soon.

Reply:
Makes sense - thanks for your feedback! Might be able to address this tomorrow (Tuesday), otherwise on Wednesday. Keep you posted!

Reply:
@escopecz Done! Thanks for the suggestion and sorry for the delay. Would you mind having another look? 😊

Reply:
There was a small bug in the code above - just fixed [here](https://github.com/mautic/mautic/pull/11382/commits/9513ac6c6af341eca59134fc206a0a6aec8fe14d). Expect CI to turn green now.